sbi/analysis/plot.py

```diff
@@ -1403,3 +1408,166 @@ def _plot_hist_region_expected_under_uniformity(
         color=color,
         alpha=alpha,
     )
+
+
+# ==== Diagnostics for L-C2ST ====
+
+
+def pp_plot_lc2st(
```

Comment:
I would not put lc2st in the function name. This function is a generic pp plot function.

Reply:
I agree. I will create a new function for general PP-plots and call it inside the `lc2st_pp_plot` function. Some things like the theoretical null values (here the dirac at 0.5) change depending on the considered test.